test/e2e/framework/resourcebinding.go

```diff
@@ -36,3 +38,63 @@ func WaitResourceBindingFitWith(client karmada.Interface, namespace, name string
 		return fit(resourceBinding)
 	}, pollTimeout, pollInterval).Should(gomega.Equal(true))
 }
+
+// UpdateResourceBindingTargetClusters update resourceBinding's target clusters
+func UpdateResourceBindingTargetClusters(client karmada.Interface, namespace, name string, clusters []workv1alpha2.TargetCluster) {
+	ginkgo.By(fmt.Sprintf("Updating ResourceBinding(%s/%s)'s target clusters to %+v", namespace, name, clusters), func() {
+		gomega.Eventually(func() error {
+			binding, err := client.WorkV1alpha2().ResourceBindings(namespace).Get(context.TODO(), name, metav1.GetOptions{})
+			if err != nil {
+				return err
+			}
+			binding.Spec.Clusters = clusters
+			_, err = client.WorkV1alpha2().ResourceBindings(namespace).Update(context.TODO(), binding, metav1.UpdateOptions{})
+			return err
+		}, pollTimeout, pollInterval).ShouldNot(gomega.HaveOccurred())
+	})
+}
+
+// AssertBindingScheduleResult wait deployment present on member clusters sync with fit func.
+func AssertBindingScheduleResult(client karmada.Interface, namespace, name string, expectedClusters []string, expectedResults [][]int32) {
+	for _, expectedResult := range expectedResults {
+		gomega.Expect(len(expectedResult)).Should(gomega.Equal(len(expectedClusters)))
+	}
+	ginkgo.By(fmt.Sprintf("Check ResourceBinding(%s/%s)'s target clusters is as expected", namespace, name), func() {
+		gomega.Eventually(func() error {
+			binding, err := client.WorkV1alpha2().ResourceBindings(namespace).Get(context.TODO(), name, metav1.GetOptions{})
+			if err != nil {
+				return err
+			}
+			scheduledResult := make([]int32, len(expectedClusters))
+			for _, scheduledCluster := range binding.Spec.Clusters {
```

Comment:
I understand that directly comparing the length of `scheduledCluster` and `expectedClusters` and then comparing with each element should be able to determine whether the results are the same.